internal/util/paramtable/base_table.go

```diff
@@ -81,7 +81,7 @@ func NewBaseTableFromYamlOnly(yaml string) *BaseTable {
 // GlobalInitWithYaml should be called only in standalone and embedded Milvus.
 func (gp *BaseTable) GlobalInitWithYaml(yaml string) {
 	gp.once.Do(func() {
-		defaultYaml = yaml
+		defaultYaml = []string{yaml}
```

Comment:
Shall we support multiple yaml files here?

Reply:
yes, keep the API simple